### PR TITLE
Documented experimental features

### PR DESCRIPTION
**Pull request by Toflar**

### CODE REVIEW

**Review by fritzmg — APPROVED** [review]

File: docs/dev/internals/experimental-features.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+annotation will be removed and the feature is now considered "stable".
+
+The following features are currently in their `@experimental` phase:
+
+* The Twig integration (`Contao\CoreBundle\Twig\*`), since Contao 4.12
+* The virtual filesystem (`Contao\CoreBundle\Filesystem\*`), since Contao 4.13
+
+{{% notice tip %}}
+If a feature is experimental, it does not mean you shouldn't be using it. Please, by all means **do use it**! Otherwise, how are
+we supposed to gather real-world feedback? The Contao core itself uses them too. The only difference to a stable API is that
```
Comment (leofeyer) [inline docs/dev/internals/experimental-features.md]:
```suggestion
we supposed to gather real-world feedback? The Contao core itself uses them, too. The only difference to a stable API is that
```

File: docs/dev/internals/experimental-features.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+annotation will be removed and the feature is now considered "stable".
+
+The following features are currently in their `@experimental` phase:
+
+* The Twig integration (`Contao\CoreBundle\Twig\*`), since Contao 4.12
+* The virtual filesystem (`Contao\CoreBundle\Filesystem\*`), since Contao 4.13
+
+{{% notice tip %}}
+If a feature is experimental, it does not mean you shouldn't be using it. Please, by all means **do use it**! Otherwise, how are
+we supposed to gather real-world feedback? The Contao core itself uses them, too. The only difference to a stable API is that
```
Comment (leofeyer) [inline docs/dev/internals/experimental-features.md]:
```suggestion
we supposed to gather real-world feedback? The Contao core itself uses it, too. The only difference to a stable API is that
```

Comment (Toflar) [reply docs/dev/internals/experimental-features.md]: The features, should be `them`?

Comment (leofeyer) [reply docs/dev/internals/experimental-features.md]: Yes, but you are only talking about "a feature", so singular. _If a feature is experimental_ … _do use it_ … _the core uses it, too_.

**Review by m-vo — APPROVED** [review]

**Review by ausi — APPROVED** [review]

**Review by leofeyer — APPROVED** [review]

**Pull request merged**